source/change-streams.rst

```diff
@@ -228,7 +228,7 @@ The driver API consists of one helper method located on a driver’s Collection
 
 The helper method must construct an aggregation command with a REQUIRED initial ``$changeStream`` stage.  A driver MUST NOT throw a custom exception if multiple ``$changeStream`` stages are present (e.g. if a user also passed ``$changeStream`` in the pipeline supplied to the helper), as the server will return an error.
 
-The aggregation command MUST use the inherited read concern of the collection it is being run against, in accordance with the `Read and Write Concern specification <https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#via-code>`_ .  The initial implementation of change streams on the server requires a “majority” read concern, drivers MUST document this requirement.  Drivers SHALL NOT throw an exception if any other read concern is specified, but instead should depend on the server to return an error.
+The aggregation command MUST use the inherited read concern of the collection it is being run against or a per-operation read concern (if supported by the driver), in accordance with the `Read and Write Concern specification <https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#via-code>`_ .  The initial implementation of change streams on the server requires a “majority” read concern, drivers MUST document this requirement.  Drivers SHALL NOT throw an exception if any other read concern is specified, but instead should depend on the server to return an error.
```

Comment:
Since this is a MUST statement, any driver which does support per-operation read concern must make that option available in collection.watch, correct?

Reply:
I'd say it's still up to drivers to decide which CRUD methods support per-operation methods. CRUD alludes to the fact that methods may take those options, but doesn't address whether it's all or nothing.

The goal here was just to ensure this spec didn't prohibit the idea of per-operation options.

Reply:
Ah ok. Up to you, but maybe reword slightly to clarify that supporting the per-operation read concern isn't necessary even if other CRUD methods support it. Something along the lines of "The aggregation command MUST use the inherited read concern of the collection it is being run against. This may be overridden by a per-operation read concern if supported by the driver."